bluefog/torch/topology_util.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, List, Optional, Union
+import collections
+
+import numpy as np
+import torch
+import bluefog.torch as bf
+
+from bluefog.common.topology_util import (
+    GetRecvWeights,
+    GetSendWeights,
+    IsRegularGraph,
+    IsTopologyEquivalent,
+)
+
+from bluefog.common.topology_util import (
+    ExponentialTwoGraph,
+    ExponentialGraph,
+    FullyConnectedGraph,
+    MeshGrid2DGraph,
+    RingGraph,
+    StarGraph,
+    SymmetricExponentialGraph,
+)
+
+from bluefog.common.topology_util import (
+    GetDynamicOnePeerSendRecvRanks,
+    GetExp2DynamicSendRecvMachineRanks,
+    GetInnerOuterRingDynamicSendRecvRanks,
+    GetInnerOuterExpo2DynamicSendRecvRanks,
+)
```

Comment:
Why importing everything here?

Reply:
Removed. Used to allow it to import by bluefog.torch.topology_util. (Not good)